js/mixins/responsiveManagerMixin.js

```diff
@@ -27,23 +28,47 @@ var ResponsiveManagerMixin = {
     this.generateResponsiveData();
   },
 
+  /**
+   * It gives the value of height, based on the width of mainVideoContainer (in a ratio 16:9);
+   * @param {number} componentWidth - width of the element
+   * @returns {number} - default height of the element
+   * If we do not have width of mainVideoContainer it returns 0
+   */
+  getDefaultElementHeight: function(componentWidth) {
```

Comment:
It would be nice to be able to pass in any ratio but default to 16:9.  Also, what about vertical video (9:16)?

Reply:
@pilievOoyala As I understood we are not able to check if the video is vertical in moments when we have to use function "getDefaultElementHeight" (for example before video starts). Could you confirm it or correct me?

Reply:
Can we use the mainVideoAspectRatio calculated here?

https://github.com/ooyala/html5-skin/blob/master/js/controller.js#L1351

Reply:
@aeng7 as I can see we pass 2 params to the function (>>calculateAspectRatio: function(width, height) {... }), but we do not know height in a start moment . That was the problem.